src/styles/index.css

```diff
@@ -105,6 +105,8 @@
     --p-color-input-checked-border: var(--p-color-selected);
     --p-color-input-checked-text: var(--p-color-text-default);
 
+    --p-color-input-checkbox-indeterminate-bg-img: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' fill='' viewBox='0 0 16 16'%3e%3cpath stroke='white' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M4 8h8'/%3e%3c/svg%3e");
```

Comment:
this svg data url is directly from tailwindcss-forms which is the current base value

https://github.com/tailwindlabs/tailwindcss-forms/blob/e6bcd1ff718ad2d0380a65ded0564369e4cbda32/src/index.js#L296

I've pulled it in so that I could recolor it for lightmode since in its current form here, the stroke is `white` which blends in with the already-white background of the checkbox in lightmode making it IMPERCEIVABLE.